types/apollo-upload-client/index.d.ts

```diff
@@ -1,22 +1,28 @@
-// Type definitions for apollo-upload-client 8.1
+// Type definitions for apollo-upload-client 14.0.0
```

Comment:
I think this will fail the linter? I don't think it likes a patch version, so should just be `14.0`